bitnami/nats/values-production.yaml

```diff
@@ -330,6 +330,25 @@ metrics:
     - -routez
     - -subz
     - -varz
+  # Enable this if you're using https://github.com/coreos/prometheus-operator
+  serviceMonitor:
+    enabled: false
+    ## Specify a namespace if needed
+    # namespace: monitoring
+    # fallback to the prometheus default unless specified
+    # interval: 10s
+    ## Defaults to what's used if you follow CoreOS [Prometheus Install Instructions](https://github.com/bitnami/charts/tree/master/bitnami/prometheus-operator#tldr)
+    ## [Prometheus Selector Label](https://github.com/bitnami/charts/tree/master/bitnami/prometheus-operator#prometheus-operator-1)
+    ## [Kube Prometheus Selector Label](https://github.com/bitnami/charts/tree/master/bitnami/prometheus-operator#exporters)
+    selector:
+      prometheus: kube-prometheus
+  service:
```

Comment:
In the case we want this service, It should be `metricService`, right?

Reply:
it's `metrics.service`?

i don't see why you would do `metrics.enabled` without enabling a service for it as well.

This is the same as all the other `metrics` or `serviceMonitor` implements across this repo

Reply:
Yes you are right!